src/unity/python/turicreate/toolkits/sound_classifier/sound_classifier.py

```diff
@@ -68,16 +76,29 @@ def create(dataset, target, feature, max_iterations=10, verbose=True, batch_size
         raise _ToolkitError("'%s' column is not audio data." % feature)
     if target not in dataset.column_names():
         raise _ToolkitError("Target column '%s' does not exist" % target)
+    if not (isinstance(validation_set, _tc.SFrame) or validation_set == 'auto' or validation_set is None):
+        raise TypeError("Unrecognized value for 'validation_set'")
```

Comment:
value -> type

Reply:
I actually think "value" makes more sense here. If the user passes in `validation_set='foobar'`, it's not the type I have a problem with, it's the value.

Reply:
That’s true, but isn’t it also the case that if I pass in an unrecognized type for the validation set, I’ll get the same error? Regardless, I don’t think this should hold up merging this in. It’s a really small nitpick anyway.

Reply:
"Value" seems fine here to me, FWIW